pkg/cloud/cloud.go

```diff
@@ -23,8 +25,10 @@ type CloudProvider interface {
 }
 
 func GetCloudProvider(
+	restConfig *rest.Config,
 	kubeClient kubernetes.Interface,
 	workClient workclient.Interface,
+	dynamicClient dynamic.Interface,
```

Comment:
Why were these 2 params added? They don't appear to be used.

Reply:
Left-overs from previous iterations where they were used, thanks for spotting them!